cmake/tools/FindMMG2D.cmake

```diff
@@ -90,9 +90,9 @@ if(MMG_INCDIR)
   find_path(MMG2D_libmmgtypes.h_DIRS
     NAMES libmmgtypes.h
     HINTS ${MMG_INCDIR}
-    PATH_SUFFIXES "mmg/mmg2d" "mmg2d")
+    PATH_SUFFIXES "mmg" "mmg/mmg2d" "mmg2d" "mmg/common")
 else()
-  if(MMG_DIR)
+    if(MMG_DIR)
```

Comment:
```suggestion
   if(MMG_DIR)
```
Unwanted change

Reply:
This should be reverted as well in https://github.com/KratosMultiphysics/Kratos/pull/10596